docs/docs/contributing/quickstart.md

```
@@ -0,0 +1,252 @@
+import Tabs from "@theme/Tabs";
+import TabItem from "@theme/TabItem";
+
+# Quickstart
+
+:::tip[Just want to _use_ the library?]
+
+If you only need to consume Img2Num (e.g. via npm), see the relevant package documentation from the links below instead. This page is for building the project itself from source.
+
+<div style={{ display: "flex", justifyContent: "space-evenly", maxWidth: "500px", margin: "0 auto" }}>
+
+  [<img src="https://cdn.jsdelivr.net/gh/devicons/devicon@latest/icons/c/c-original.svg" width="30" />](../../c)
```

markdownlint-cli2 (0.22.1)

[warning] Images should have alternate text (alt text)

(MD045, no-alt-text)